src/components/Navbar/Navbar.jsx

```diff
@@ -1,15 +1,24 @@
-import React from "react";
+import React, { useState } from "react";
 import { NavLink } from "react-router-dom";
 import Button from "../Button";
 import "./Navbar.css";
 
-const Navbar = ({ handleTabChange }) => {
+const Navbar = ({ handleTabChange, handleSearch }) => {
+  const [searchTerm, setSeachTerm] = useState("");
```

Comment:
Should we need two states for holding searchtext, it may cause data redundancy.

Data redundancy is the practice of storing the same data in multiple locations, either within a single database or across multiple databases.

Reply:
We are storing the same search text data in app.js page also.